src/test/kotlin/pt/up/fe/ni/website/backend/controller/AccountControllerTest.kt

```diff
@@ -373,4 +396,22 @@ class AccountControllerTest @Autowired constructor(
         // objectMapper adds quotes to the date, so remove them
         return quotedDate.substring(1, quotedDate.length - 1)
     }
+
+    fun Account?.toJson(): String {
+        // password is ignored on serialization, so add it manually
+        // for account creation test cases
+        return objectMapper.writeValueAsString(
+            mapOf(
+                "name" to this?.name,
+                "email" to this?.email,
+                "password" to this?.password,
+                "bio" to this?.bio,
+                "birthDate" to this?.birthDate.toJson(),
+                "photoPath" to this?.photoPath,
+                "linkedin" to this?.linkedin,
+                "github" to this?.github,
+                "websites" to this?.websites
+            )
```

Comment:
```suggestion
            objectMapper.convertValue(this, Map::class.java).plus(
                mapOf(
                    "password" to this?.password
                )
            )
        )
```
This seems better to mantain

Reply:
Awesome!